admin/src/components/AuthService.js

```diff
@@ -0,0 +1 @@
+export const getToken = () => window.localStorage.getItem("token");
```

Comment:
Ez jó lesz, de azért figyelj más branchekre is, nehogy ugyanaz elkészüljön többször: https://github.com/flowacademyhu/piac-app/blob/feature-admin-edit-market/admin/src/components/AuthService.js